src/components/TransactionConfirmationModal/index.tsx

```diff
@@ -173,16 +173,27 @@ export function ConfirmationModalContent({
   bottomContent,
   onDismiss,
   topContent,
+  chainId,
 }: {
   title: ReactNode
   onDismiss: () => void
   topContent: () => ReactNode
   bottomContent?: () => ReactNode | undefined
+  chainId?: SupportedL2ChainId | undefined
 }) {
+  const info = getChainInfo(chainId)
   return (
     <Wrapper>
       <AutoColumn gap="sm">
         <Row>
+          {chainId && (
```

Comment:
You should check for `{info &&}` instead of `chainId`, and then, inside the component, do not write `{info?.label}` but just `{info.label}`. We don't want any undefined.